File: tools/dbench/answers/go_haiku/s2/f2_adults.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	f, _ := os.Open("users.txt")
	defer f.Close()

	sc := bufio.NewScanner(f)
	var adults []string

	for sc.Scan() {
		parts := strings.Fields(sc.Text())
		if len(parts) != 3 {
			continue
		}
		age, err := strconv.Atoi(parts[2])
		if err != nil {
			continue
		}
		if age > 25 {
			adults = append(adults, parts[1])
		}
	}

	fmt.Println(strings.Join(adults, " "))
}

File: tools/dbench/answers/python_sonnet/s1/f2_adults.py
adults = []
with open("users.txt") as f:
    for line in f:
        parts = line.split()
        if len(parts) == 3:
            try:
                if int(parts[2]) > 25:
                    adults.append(parts[1])
            except ValueError:
                pass

print(" ".join(adults))
